plugins/hosts/darwin/cap/rdp.rb

```diff
@@ -40,12 +40,12 @@ def self.generate_config_file(rdp_info)
             f.binmode
 
             opts.each do |k, v|
-              f.write("#{k}:#{v}")
+              f.write("#{k}:#{v}\r\n")
```

Comment:
It's an RDP file. The version of Microsoft Remote Desktop I have (latest from App Store) doesn't care...

The code above it does `f.binmode` just above, so I thought someone was being paranoid and went with it.